crates/flowctl/src/catalog/mod.rs

```diff
@@ -80,7 +80,7 @@ pub enum Command {
 #[derive(Default, Debug, Clone, clap::Args)]
 pub struct NameSelector {
     /// Select a spec by name. May be provided multiple times.
-    #[clap(long)]
+    #[clap(long, required = true)]
```

Comment:
These arguments are also used by lots of other subcommands, and I don't think we want to require them in all contexts. I'm thinking we should figure out a way for one of these to only be required if the sucommand is `delete`. That _might_ need to be an explicit check inside of `do_delete`, if there isn't something like `required_if_subcommand_eq` in clap. Or I suppose we could add a separate `DeleteNameSelector` that requires on of these args and implements `Into<NameSelector>`.